sdk/keyvault/azure-security-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretClientBuilder.java

```diff
@@ -61,7 +62,7 @@
     private final List<HttpPipelinePolicy> policies;
     private TokenCredential credential;
     private HttpPipeline pipeline;
-    private URL endpoint;
```

Comment:
Need to change `endpoint` in the above docs to `vaultUrl`

Reply:
updated